src/mlpack/methods/elm/elm.hpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file elm.hpp
+ * @author Siddharth Agrawal
+ *
+ * Basic Extreme Learning Machine
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_ELM_HPP
+#define MLPACK_METHODS_ELM_HPP
+
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+namespace elm {
+
```

Comment:
Please add comment at here, explain the purpose of this class and give some example. You can take [softmax](https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/softmax_regression/softmax_regression.hpp) as example.